src/panels/config/integrations/integration-panels/thread/thread-config-panel.ts

```diff
@@ -151,7 +151,7 @@ export class ThreadConfigPanel extends SubscribeMixin(LitElement) {
               slot="fab"
               @click=${this._importExternalThreadCredentials}
               extended
-              label="Import credentials"
+              label="Send credentials to Home Assistant"
```

Comment:
This should be localized, so it can be translated into other languages.

Reply:
Ok, but there are several strings untranslated it seems currently. Should we maybe convert these in one go? :thinking:

Reply:
It should be OK to do this in one go (in a different PR), since your other PR in this area also has some untranslatable sentences that should possibly be localized.